### PR TITLE
Add pre-trained weights for ConvMixer_512_16

### PR DESCRIPTION
**Pull request by ianstenbit**
These score 74.38% top-1 accuracy, better than the paper-claimed 73.76%

### CODE REVIEW

**Comment by ianstenbit** [comment]
Sidebar: I screwed up my commit history here -- sorry about that. Squashing should make it a non-issue

**Comment by IMvision12** [comment]
@ianstenbit aren't we going to add weights for other variants? Because as per the paper ConvMixer-1536/20 variants have an accuracy of 82.20%

**Comment by ianstenbit** [comment]
> @ianstenbit aren't we going to add weights for other variants? Because as per the paper ConvMixer-1536/20 variants have an accuracy of 82.20% it could be nice if we can add weights for this variant too.

Yes, we can add weights for other variants in the future. For now, I am adding these because I used the smallest variant to verify the numerics of the implementation

**Comment by IMvision12** [comment]
> > @ianstenbit aren't we going to add weights for other variants? Because as per the paper ConvMixer-1536/20 variants have an accuracy of 82.20% it could be nice if we can add weights for this variant too.
> 
> Yes, we can add weights for other variants in the future. For now, I am adding these because I used the smallest variant to verify the numerics of the implementation

Okay

**Review by LukeWood — APPROVED** [review]
